lib/modules/manager/bundler/artifacts.spec.ts

```diff
@@ -967,6 +967,28 @@ describe('modules/manager/bundler/artifacts', () => {
         expect(res).toMatchObject([{ file: { path: 'Gemfile.lock' } }]);
         expect(execSnapshots).toMatchObject([{ cmd: 'bundler lock' }]);
       });
+
```

Comment:
Hi @benlangfeld can you explain in the PR body text _why_ you're making this change, and link to relevant issues/discussions, and so on?